packages/react-kit/src/stories/widgets/Commit.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { ComponentMeta, ComponentStory, Story } from "@storybook/react";
+
+import { CommitWidget } from "../../components/widgets/commit/CommitWidget";
+import { EnvironmentType, getEnvConfigs } from "@bosonprotocol/core-sdk";
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: "Widgets/Commit",
+  component: CommitWidget
+} as ComponentMeta<typeof CommitWidget>;
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Template: ComponentStory<typeof CommitWidget> = (args) => (
+  <CommitWidget {...args} />
+);
+
+const wrapper = (Story: Story) => (
+  <div>
+    <Story />
+  </div>
+);
+
+export const Commit: ComponentStory<typeof CommitWidget> = Template.bind({});
+
+const envName =
+  (process.env.STORYBOOK_DATA_ENV_NAME as EnvironmentType) || "testing";
+const envConfig = getEnvConfigs(envName);
+// More on args: https://storybook.js.org/docs/react/writing-stories/args
+Commit.args = {
+  envName,
+  configId: envConfig[0].configId,
+  walletConnectProjectId: process.env.REACT_APP_WALLET_CONNECT_PROJECT_ID,
+  dateFormat: "YYYY/MM/DD",
+  defaultCurrencySymbol: "$",
+  defaultCurrencyTicker: "USD",
+  contactSellerForExchangeUrl: "https://bosonapp.io/#/chat/{id}",
+  fairExchangePolicyRules:
+    "ipfs://QmV3Wy2wmrFdEXzhyhvvaW25Q8w2wTd2UypFVyhwsdBE8T",
+  ipfsGateway: process.env.STORYBOOK_DATA_IPFS_GATEWAY,
+  ipfsProjectId: process.env.STORYBOOK_DATA_IPFS_PROJECT_ID,
+  ipfsProjectSecret: process.env.STORYBOOK_DATA_IPFS_PROJECT_SECRET,
+  offerId: "",
+  defaultSelectedOfferId: "",
+  disableVariationsSelects: false,
+  productUuid: "086b32-3fcd-00d1-0624-67513e85415c", // with size variations
+  // productUuid: "1d4573d-42e-f557-0032-b1ab8170c102", // with color and size variations
```

Comment:
create other stories with this other parameters
(see https://github.com/bosonprotocol/core-components/blob/a440bc4b01fe021d05334524c1e90429a439b23c/packages/react-kit/src/stories/widgets/Redemption.stories.tsx#L67C1-L67C1 as an example)